src/pages/StartContext.tsx

```diff
@@ -69,9 +68,9 @@ export default function StartContextPage() {
     setShowStatusModal(true);
   };
 
-  const installApplicationHandler = async (): Promise<boolean> => {
+  const installApplicationHandler = async (): Promise<string> => {
     if (!application.appId || !application.version) {
-      return false;
+      return '';
```

Comment:
What do we do with '' ?

Reply:
can do null, in other function it checks if appId is "" then returns

Reply:
Updated